vault-server/docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: "3.3"
+services:
+  consul-server:
+    build: ./consul
+    image: consul:1.12.2 
+    restart: always
+    networks:
+      - consul
+    command: "consul agent -config-dir /etc/consul.d"
+    container_name: consul-server
+    volumes:
+      - ./consul/consul-server.json:/etc/consul.d/consul-server.json:rw
+      - ./consul-certs/:/etc/consul.d/certs/:ro
+    ports:
+      - "8300:8300"
```

Comment:
a lot of ports are opened now

Reply:
I corrected the items that you mentioned